tests/snapshots/assets__check_file@unit-code-short-if.typ-40.snap

```diff
@@ -0,0 +1,32 @@
+---
+source: tests/assets.rs
+expression: doc_string
+input_file: tests/assets/unit/code/short-if.typ
+snapshot_kind: text
+---
+#table(
+  fill: (x, y) => if y == 0 {
+    white.darken(15%)
+  } else { none },
+  align: (x, y) => if y == 0 {
+    center
+  } else { horizon },
+  [Hi],
+  [there],
+)
```

Comment:
This doesn't look nice. This is hard to implement in wadler style pretty printer. I always want to control the display style (inline or not) of A depending on B's state. But wadler style pretty printer simply doesn't support that. I guess eventually we need to propose a new IR for pretty printer and implement it by our own.